.scalafmt.conf

```diff
@@ -2,10 +2,36 @@ version = 3.5.3
 style = defaultWithAlign
 maxColumn = 100
 assumeStandardLibraryStripMargin = true
-runner.dialect = "scala213source3"
+runner.dialect = scala213source3
 
 project.excludeFilters = [
    "scalapb-runtime/src/main/js-native/com/google/protobuf/.*",
    "scalapb-runtime/.*/scala.*/com/google/protobuf/.*",
    "scalapb-runtime/.*/scala.*/scalapb/options/.*"
 ]
+fileOverride {
```

Comment:
is this needed? generated files are not subject to scalafmt.

Reply:
Metals requested to update .scalafmt.conf accordingly. Maybe it may be ignored, but seems to me this will not do any harm.

If you're against - we may rollback it

Reply:
My preference is to keep the PRs focused only on the changes necessary for the issue, and send changes like this as a separate PR. It would be great if you can roll back this file.

In the mean time how can I reproduce metals adding this locally?

Reply:
I've just run "Import build" in VSCode

<img width="466" alt="изображение" src="https://user-images.githubusercontent.com/31408417/169914747-460734cf-355a-49c8-b01f-145ddabc1461.png">

Reply:
Nevermind, I've chosen "don't show again" xD